pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1385,7 +1385,9 @@ public void addTable(TableConfig tableConfig)
       throws IOException {
     String tableNameWithType = tableConfig.getTableName();
     if (getTableConfig(tableNameWithType) != null) {
-      throw new TableAlreadyExistsException("Table " + tableNameWithType + " already exists");
+      throw new TableAlreadyExistsException("Table config for " + tableNameWithType
+          + " already exists. If this is unexpected, try deleting the table to remove all metadata associated"
+          + " with it.");
```

Comment:
qq: if table config already exist. why can't we use the PUT endpoint to modify the streamConfig so that ingestion error could be fixed?

any reason we had to delete and re-create again? (is it because the table isn't showing up on UI?)